mintpy/smallbaselineApp.py

```diff
@@ -769,6 +772,19 @@ def get_dataset_size(fname):
                     mintpy.tropo_phase_elevation.main(iargs)
 
             # Weather Re-analysis Data (Jolivet et al., 2011;2014)
+            elif method == 'gacos':
+                from mintpy.tropo_pyaps3 import correct_timeseries
+                import glob
+                tropo_file = self.workDir + '/GACOS.h5'
+                GACOS_dir = self.template['mintpy.troposphericDelay.gacosDir']
+                iargs = [in_file, '-l', geom_file, '-i', geom_file, '-o', tropo_file, '--GACOS-dir', GACOS_dir]
+                print('tropospheric delay correction with gacos approach')
+                print('\ntropo_gacos.py', ' '.join(iargs))
+                if ut.run_or_skip(out_file=out_file, in_file=in_file) == 'run':
+                    mintpy.tropo_gacos.main(iargs)
+                    correct_timeseries(in_file, tropo_file, out_file)
+
```

Comment:
The `mintpy.tropo_pyaps3.correct_timeseries()` is a very short function, I would recommend to copy the code from tropo_pyaps3.py into tropo_gacos.py so that 1) the import here can be simplified and 2) without import tropo_pyaps3.py, which checks pyaps3.

Reply:
For `mintpy.tropo_gacos.main()` to work, you might need to add it work the `workflow/__init__.py`.